docroot/themes/custom/mass_theme/templates/paragraphs/paragraph--what-would-you-like-to-do.html.twig

```diff
@@ -46,7 +46,6 @@ solution will need a refactor. This ID is currently being used as an
 anchor link in field--extra-org-nav-featured-services.html.twig. #}
     {% embed "@organisms/by-author/action-finder.twig" with {
       "actionFinder": {
-        "id": "what-would-you-like-to-do",
```

Comment:
I think removing this will break a link in the SubNav bar based on the comments above the embed code.

Reply:
yes, you are right, just added that back. At some point this should be refactored so the id is unique if multiple paragraphs are added.